libs/sdk-model/src/execution/results/index.ts

```diff
@@ -111,6 +111,26 @@ export interface ITotalDescriptor {
     totalHeaderItem: ITotalDescriptorItem;
 }
 
+/**
+ * Describes color included in a dimension.
+ *
+ * @public
+ */
+export interface IColorDescriptorItem {
+    id: string;
+    name: string;
+}
+
+/**
+ * Describes color included in a dimension.
+ *
+ * @public
+ */
+export interface IColorDescriptor {
+    // TODO: rename this to colorDescriptor ... the goal is to get rid of the overused 'header' nomenclature
```

Comment:
Please check with @kandl what is the vision of these changes, whether we should keep the TODO note here

Reply:
I updated this. Thank you!